Apps/VisionOS/VisionOS/ViewUtils/InlineNavigationLink.swift

```diff
@@ -22,4 +22,49 @@ enum InlineNavigationLink: String, Codable, CaseIterable {
 
         self = link
     }
+
+    var menuTitle: String {
+        // TODO: menu title for each screen
+        switch self {
+        case .sampleAppIntro:
+            ""
+        case .install:
+            ""
+        case .setup:
+            ""
+        case .customerIOIntro:
+            ""
+        case .identify:
+            ""
+        case .howToTestIdentify:
+            ""
+        case .profileAttributes:
+            ""
+        case .howToTestProfileAttributes:
+            ""
+        case .deviceAttributes:
+            ""
+        case .howToTestDeviceAttributes:
+            ""
+        case .track:
+            ""
+        case .howToTestTrack:
+            ""
+        }
+    }
```

Comment:
Non-blocking question: Is it possible for each screen's View to decide what the menu title is? I am OK with this if that's preferred.

Reply:
You are spot on. Each of these cases will get the title from the relevant screen. Hence it is currently blank "to indicate the missing title". The only reason this is added here as an associated property, is to avoid the need for each callsite to do the logic of this switch case.